### ISSUE TITLE
Explaination about account deletion

### ISSUE THREAD

**Issue by OlivierCollin** [genouest/genouestaccountmanager#170]
When deleting dubious user accounts, it is not a good policy to send an email. 

For example, I just deleted a pending email approval of an account that was not legitimate (gmail account, suspicious infos, etc.) 
I do not see the point of providing any explanations, including my user login, for such a request. 
I even saw a "no explanation provided ! " alert... 
I do not want to enter into a negociation with someone not playing by the rules, or even trying to hack our systems. 

At least we should be able to deactivate this functionality in such cases.

**Comment by FanchTheSystem** [comment]
Hello,
It is not well documented

but I think that if you leave the explanation blank, no email will be sent to the account deleted.

**Comment by FanchTheSystem** [comment]
see: https://github.com/genouest/genouestaccountmanager/blob/92ff5916a37fca566bf509ffccf326ce68f4105b/routes/users.js#L824-L828

**Comment by osallou** [comment]
@OlivierCollin do you confirm?

**Comment by FanchTheSystem** [comment]
yes I will add soon a checkbox to send or not the email (ifb and abims ask for it too)
